plugins/reach/lib/model/EntryVendorTask.php

```diff
@@ -252,6 +332,11 @@ public function preSave(PropelPDO $con = null)
 				{
 					$turnAroundTime = self::SEVEN_DAYS;
 				}
+				elseif( ($turnAroundTime == VendorServiceTurnAroundTime::ONE_BUSINESS_DAY) ||
```

Comment:
Add up to 7 business days.
If the value is between 1 and max days, send the value , so we can support more days then  1 & 2